src/lib/import_export/binary/binary_writer.hpp

```diff
@@ -125,12 +125,35 @@ class BinaryWriter {
    * ^: These fields are only written if the type of the column IS a string.
    * °: This field is written if the type of the column is NOT a string
    *
-   * @param base_dictionary_segment The segment to export
+   * @param dictionary_segment The segment to export
    * @param ofstream The output stream for exporting
    */
   template <typename T>
   static void _write_segment(const DictionarySegment<T>& dictionary_segment, std::ofstream& ofstream);
 
+  /**
+   * FixedStringDictionary Segments are dumped with the following layout:
+   *
+   * Description           | Type                                  | Size in bytes
+   * -----------------------------------------------------------------------------------------
+   * Column Type           | ColumnType                            |   1
```

Comment:
Below, you say "The type of the column can be found in the global header of the file."

Don't you mean EncodingType? Same for other encodings.